ui/src/timeMachine/selectors/index.ts

```diff
@@ -249,3 +250,75 @@ export const getSaveableView = (state: AppState): QueryView & {id?: string} => {
 
   return saveableView
 }
+
+export const getStartTime = timeRange => {
+  const {lower} = timeRange
+  switch (lower) {
+    case 'now() - 30d':
+      return (
+        moment()
+          .subtract(30, 'days')
+          .unix() * 1000
+      )
+    case 'now() - 7d':
+      return (
+        moment()
```

Comment:
I am worried about when this moment() will be instantiated..... doing some tests.